aiohttp/connector.py

```diff
@@ -1126,3 +1127,53 @@ def path(self) -> str:
             raise ClientConnectorError(req.connection_key, exc) from exc
 
         return cast(ResponseHandler, proto)
+
+
+class NamedPipeConnector(BaseConnector):
+    """Named pipe connector.
+    Only supported by the proactor event loop.
+    See also: https://docs.python.org/3.7/library/asyncio-eventloop.html
+
+    path - Windows named pipe path.
+    keepalive_timeout - (optional) Keep-alive timeout.
+    force_close - Set to True to force close and do reconnect
+        after each request (and between redirects).
+    limit - The total number of simultaneous connections.
+    limit_per_host - Number of simultaneous connections to one host.
+    loop - Optional event loop.
+    """
+
+    def __init__(self, path: str, force_close: bool=False,
+                 keepalive_timeout: Union[object, float, None]=sentinel,
+                 limit: int=100, limit_per_host: int=0,
+                 loop: Optional[asyncio.AbstractEventLoop]=None) -> None:
+        super().__init__(force_close=force_close,
+                         keepalive_timeout=keepalive_timeout,
+                         limit=limit, limit_per_host=limit_per_host, loop=loop)
+        self._path = path
+
+    @property
+    def path(self) -> str:
+        """Path to the named pipe."""
+        return self._path
+
+    async def _create_connection(self, req: 'ClientRequest',
+                                 traces: List['Trace'],
+                                 timeout: 'ClientTimeout') -> ResponseHandler:
+        try:
+            with CeilTimeout(timeout.sock_connect):
+                _, proto = \
```

Comment:
Please, use brackets instead of EOL escaping
```suggestion
                _, proto = (
```